rs/rosetta-api/icp_ledger/rosetta-integration-tests/tests/tests.rs

```diff
@@ -402,14 +429,14 @@ async fn test_rosetta_blocks_mode_enabled() {
 
     // Check that restarting Rosetta doesn't enable the
     // rosetta blocks mode
-    env.restart_rosetta_node(false).await;
+    env.restart_rosetta_node(false, true).await.unwrap();
     assert_eq!(
         env.rosetta.status_or_panic().await.rosetta_blocks_mode,
         RosettaBlocksMode::Disabled
     );
     // Check that restarting Rosetta doesn't enable the
     // rosetta blocks mode
-    env.restart_rosetta_node(false).await;
+    env.restart_rosetta_node(false, true).await.unwrap();
```

Comment:
IIUC, this restarts Rosetta twice, and both times verifies the same thing. What's the purpose of doing it twice?

Reply:
To see whether rosetta blocks stays disabled when restarting it.